clang/include/clang/Basic/DiagnosticParseKinds.td

```diff
@@ -215,6 +215,8 @@ def warn_c23_compat_case_range : Warning<
   DefaultIgnore, InGroup<CPre2yCompat>;
 def ext_c2y_case_range : Extension<
   "case ranges are a C2y extension">, InGroup<C2y>;
+def err_c2y_labeled_break_continue
+    : Error<"labeled %select{'break'|'continue'}0 is only supported in C2y">;
```

Comment:
```suggestion
def err_c2y_labeled_break_continue : Error<
  "labeled %select{'break'|'continue'}0 is only supported in C2y">;
```

Reply:
I believe that was `clang-format` and not me.

Reply:
I typically just run `git clang-format`, so CI might complain if I don’t format it like this but I’ll try reformating it and see if it does.